src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
+    <h1>{{ userName }}</h1>
+    <div id="totalCount">
+      <p>{{ totalRepoCount }} repositories</p>
+      <p>|</p>
+      <p>{{ totalRepoStarCount }} stars</p>
+    </div>
+    <hr>
+    <p v-if="isEmptyRepoList">
+      레포지토리가 존재하지 않습니다.
+    </p>
+    <repo-list-item
+      v-else
+      v-for="(repoListItem,idx) in repoList"
+      :key="idx"
+      :repo-name="repoListItem.name"
+      :repo-star="repoListItem.stargazers_count"
+    />
+  </div>
+</template>
+
+<script>
+import RepoListItem from './RepoListItem.vue';
+
+export default {
+  name: 'RepoList',
+  components: {
+    RepoListItem,
+  },
+  props: {
+    userName: {
+      type: String,
+      default: '',
+    },
+  },
+  data() {
+    return {
+      repoList: [],
+    };
+  },
+  computed: {
+    isEmptyRepoList() {
+      return !this.repoList;
+    },
+    totalRepoCount() {
+      return this.repoList.length;
+    },
+    totalRepoStarCount() {
+      return this.repoList
+        .map((element) => element.stargazers_count, [])
+        .reduce((x, y) => x + y, 0);
```

Comment:
정말 사소한 개인 피드백인데
`reduce`의 경우엔 js에서 기본적으로 내장된 함수여서 다른 분들이 쉽게 params 값들을 유추 할 수있을 듯 하나,
그럼에도 불구하고 최소한의 의미 부여를 하는 것은 어떻게 생각하세요?


```suggestion
        .reduce((accumulator, currentValue) => accumulator + currentValue, 0);
```

Reply:
좋은 방법이라고 생각해요!

Reply:
좋은거 같아요! 수정하도록 하겠습니다 ㅎㅎ